getting-started/multiband-analysis.rst

```diff
@@ -291,7 +319,7 @@ Here are some takeaways:
 - Use the ``base_ClassificationExtendedness_value`` column to quickly distinguish stars from galaxies.
 - The ``base_PsfFlux_flag`` column is useful for identifying sources that don't have photometric measurement errors.
 
-In the end, you created a simple *r-i* CMD.
+In the end, you created a simple color-color diagram.
 This tutorial is just the beginning, though.
```

Comment:
Is it worth mentioning the object tables? I thought we were hoping that they would be the user facing tables in the end.

Reply:
I'm going to leave this as is for the moment.  I'll make a ticket to move off using the `src` products.  You are definitely correct we should be moving to using the object tables